### PR TITLE
Add links to next and previous episodes

### PR DESCRIPTION
**Pull request by stephaniehobson**
mozilla/irlpodcast#82

### CODE REVIEW

File: src/scss/irlpodcast.scss [hunk 1]
```diff
@@ -626,6 +626,48 @@ a.episode-shownotes-link {
     }
 }
 
+.next {
+    &:before {
```
Comment (jpetto) [inline src/scss/irlpodcast.scss]: Nit - extra nesting. Could just be `.next:before`.

File: src/scss/irlpodcast.scss [hunk 2]
```diff
@@ -626,6 +626,48 @@ a.episode-shownotes-link {
...
+        float: left;
+    }
+
+    .next-prev {
+        padding-right: 20px;
+    }
+
+    .next-next {
+        float: right;
+        padding-left: 20px;
```
Comment (jpetto) [inline src/scss/irlpodcast.scss]: I think the UI could use a little attention for skinnier viewports.

When `< $mq-phone-wide`, the left/right alignment looks a little broken.

screen shot 2018-08-28 at 3 12 13 pm

When `>= $mq-phone-wide`, the alignment is also a little broken if the titles are long-ish.

screen shot 2018-08-28 at 3 13 58 pm

Maybe just left align them until `$mq-phone-wide` (or `$mq-tablet`?) and then set a max width when floating?

**Comment by stephaniehobson** [comment]
Awww, I thought my "drops to the next line when the title is too long" solution was clever :P

I'll look into containing the longer titles.

**Comment by jpetto** [comment]
Aw, I didn't mean to denigrate! 😞 

I think it's just because the "next" content is the only thing right aligned on the page. Also, I am not a designer _or_ UX expert. My opinion does not carry that much weight. 😬

**Comment by stephaniehobson** [comment]
It's it not clear to the second person who looks at it what's going on it's clearly not that clever a solution 😉

**Comment by stephaniehobson** [comment]
Updated ⬅️

File: src/scss/irlpodcast.scss [hunk 3]
```diff
@@ -626,6 +626,45 @@ a.episode-shownotes-link {
     }
 }
 
+.next:before {
+        @include gradient-bar();
+    }
+}
```
Comment (jpetto) [inline src/scss/irlpodcast.scss]: Extra end curly brace here 🙅‍♂️

**Comment by stephaniehobson** [comment]
Curly brace ✂️

**Pull request merged**